command/init.go

```diff
@@ -337,6 +337,15 @@ job "example" {
       #   change_signal = "SIGHUP"
       # }
 
+      # The "template" stanza can also be used to create env vars for tasks
```

Comment:
We should probably make it a validation error to use signal and env at the same time.